bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/DiscoveryResultImpl.java

```diff
@@ -215,7 +228,16 @@ public boolean equals(Object obj) {
     @Override
     public String toString() {
         return "DiscoveryResult [thingUID=" + thingUID + ", properties=" + properties + ", flag=" + flag + ", label="
-                + label + ", bridgeUID=" + bridgeUID + "]";
+                + label + ", bridgeUID=" + bridgeUID + ", ttl=" + timeToLive + ", createdAt=" + createdAt + "]";
+    }
+    
+    @Override
+    public long getCreatedAt() {
```

Comment:
"created" is probably not fully correct - isn't this also updated, if a result entry is updated?